Translated/Dialog/Dighal/ssn_011110.csv

```diff
@@ -0,0 +1,13 @@
+ssn_011110_0001#0,"""Have you tried your hand at Gathering?"""
+ssn_011110_0002#0,"""Outside the city, there's a wealth of vegetables, fruits, Ores and Tames meat to be harvested."""
+ssn_011110_0003#0,"""Those fresh ingredients and strong materials are sure to benefit you in your battles to come."""
+ssn_011110_0004#0,"""Just something to think about. Take it easy!"""
+ssn_011110_0101#0,"""How's your Gathering been going?"""
+ssn_011110_0102#0,"""Personally, I like mining. Whenever I get some free time, I often use it to canvas a region for its Ores."""
```

Comment:
Mentioned this in Discord but "Whenever... I often..." is a weird sentence construction because it's essentially saying "When x happens, I always sometimes do y".